### DISCUSSION TITLE
CLI для деплоя и сборки проектов Code

### DISCUSSION THREAD

**Discussion by snyuryev** [sberdevices/salute-issues#185] [SmartMarket Studio]
Хотелось бы редактировать и деплоить проекты Code прямо из IDE (например VS Code).

**Comment by Genenko-Svetlana** [comment]
@skobets  @asmalakhov-sberdevices тема для обсуждения

**Comment by awinogradov** [comment]
Кмк, лучше научить Code работать с выделенным remote на git-репозитории, через подписку на пуш. Тогда git автоматически такой CLI. Так, например, работает heroku
